### DISCUSSION TITLE
[HARDWARE ISSUE] Pico showing held buttons

### DISCUSSION THREAD

**Discussion by SAlluBe** [OpenStickCommunity/GP2040-CE#673] [Q&A]
i made a Leverless controler for myself and i used pico just like you but when open game controler setting in windows there were some buttons are pressed. load the Tekken 7 to confirm and its giving commands on its own. i checked my soldring and wiring nothing was wrong so i thought pico is broken or faulty. i changed pico. solder the new one plugged in same problem. change 3 data cables but no success can you tell me what could be the problem or how i can fix this issue ?

**Comment by TheTrainGoes** [comment]
Hi @SAlluBe ,

This sounds like a soldering issue or something that is bridging to ground.  Can you please upload some pics of your wiring and soldering?

**Comment by TheTrainGoes** [comment]
Closing due to lack of reply.
